### PR TITLE
Add async example + GitHub action + README update

### PR DESCRIPTION
**Pull request by robsdedude**

### CODE REVIEW

File: README.adoc [hunk 1]
```diff
@@ -22,7 +27,9 @@ Provision a database quickly with https://sandbox.neo4j.com/?usecase=movies[Neo4
 
 === Setup
 
-First get yourself setup with link:http://docs.python-guide.org/en/latest/dev/virtualenvs/[virtualenv] so we don't break any other Python stuff you have on your machine. After you've got that installed let's setup an environment for our app:
+Install Python 3.6-3.9 (sync) or 3.7-3.10 (async).
+
+Then get yourself setup with link:http://docs.python-guide.org/en/latest/dev/virtualenvs/[virtualenv] so we don't break any other Python stuff you have on your machine. After you've got that installed let's setup an environment for our app:
```
Comment (fbiville) [inline README.adoc:32]:
```suggestion
Then get yourself set up with link:http://docs.python-guide.org/en/latest/dev/virtualenvs/[virtualenv] so we don't break any other Python stuff you have on your machine. After you've got that installed let's set up an environment for our app:
```

File: movies_async.py [hunk 2]
```diff
@@ -0,0 +1,161 @@
...
+        updates = summary.counters.properties_set
+
+        return {"updates": updates}
+
+
+if __name__ == "__main__":
+    import uvicorn
+
+    logging.root.setLevel(logging.INFO)
+    logging.info("Staring on port %d, database is at %s", port, url)
```
Comment (fbiville) [inline movies_async.py:159]:
```suggestion
    logging.info("Starting on port %d, database is at %s", port, url)
```

**Review by fbiville — APPROVED** [review]
just a few typos to fix

File: README.adoc (hunk 1 above)
Comment (robsdedude) [reply README.adoc:32]: :D was there before. Good you caught it now.

**Review by fbiville — APPROVED** [review]

**Pull request merged**